<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1042</width>
        <height>533</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>MainWindow</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
          <rect>
            <x>590</x>
            <y>240</y>
            <width>431</width>
            <height>51</height>
          </rect>
        </property>
        <layout class="QHBoxLayout" name="color_horizontalLayout">
          <item>
            <widget class="QLineEdit" name="textColorCode">
              <property name="readOnly">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="buttonPickScreen">
              <property name="text">
                <string>Pick Color</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="buttonColorPalette">
              <property name="text">
                <string>Color Palette</string>
              </property>
            </widget>
          </item>
        </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
          <rect>
            <x>20</x>
            <y>10</y>
            <width>561</width>
            <height>461</height>
          </rect>
        </property>
        <layout class="QVBoxLayout" name="video_mainVBoxLayout">
          <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
            <widget class="QLabel" name="video_labelInput">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Input Video</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QVideoWidget" name="video_displayWidget" native="true"/>
          </item>
          <item>
            <layout class="QHBoxLayout" name="video_toolHBoxLayout">
              <property name="sizeConstraint">
                <enum>QLayout::SetMinimumSize</enum>
              </property>
              <item>
                <widget class="QPushButton" name="video_buttonOpen">
                  <property name="text">
                    <string>Open</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="video_buttonPlay">
                  <property name="enabled">
                    <bool>true</bool>
                  </property>
                  <property name="text">
                    <string>Play</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="video_labelCurrent">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>0:00</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QSlider" name="video_hSlider">
                  <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="video_labelTotal">
                  <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string/>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <widget class="QLabel" name="video_labelStatus">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Status</string>
              </property>
            </widget>
          </item>
        </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
          <rect>
            <x>590</x>
            <y>300</y>
            <width>431</width>
            <height>171</height>
          </rect>
        </property>
        <layout class="QGridLayout" name="edit_gridLayout">
          <item row="0" column="2">
            <widget class="QLabel" name="edit_labelTolValue">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>    0 %</string>
              </property>
            </widget>
          </item>
          <item row="0" column="0">
            <widget class="QLabel" name="edit_labelTol">
              <property name="text">
                <string>Tolerance</string>
              </property>
            </widget>
          </item>
          <item row="2" column="1">
            <widget class="QSlider" name="edit_hSliderDefringe">
              <property name="maximum">
                <number>100</number>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </widget>
          </item>
          <item row="2" column="0">
            <widget class="QLabel" name="edit_labelDefringe">
              <property name="text">
                <string>Defringe</string>
              </property>
            </widget>
          </item>
          <item row="0" column="1">
            <widget class="QSlider" name="edit_hSliderTol">
              <property name="maximum">
                <number>100</number>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </widget>
          </item>
          <item row="1" column="2">
            <widget class="QLabel" name="edit_labelSoftValue">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>    0 %</string>
              </property>
            </widget>
          </item>
          <item row="1" column="0">
            <widget class="QLabel" name="edit_labelSoft">
              <property name="text">
                <string>Softness</string>
              </property>
            </widget>
          </item>
          <item row="1" column="1">
            <widget class="QSlider" name="edit_hSliderSoft">
              <property name="maximum">
                <number>100</number>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </widget>
          </item>
          <item row="2" column="2">
            <widget class="QLabel" name="edit_labelDefringeValue">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>    0 %</string>
              </property>
            </widget>
          </item>
          <item row="3" column="1">
            <widget class="QPushButton" name="edit_buttonSave">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Save</string>
              </property>
            </widget>
          </item>
        </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
          <rect>
            <x>590</x>
            <y>10</y>
            <width>351</width>
            <height>221</height>
          </rect>
        </property>
        <layout class="QVBoxLayout" name="video_outputLayout">
          <item>
            <widget class="QLabel" name="video_labelOutput">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Output Video</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLabel" name="video_outputFrame">
              <property name="text">
                <string/>
              </property>
            </widget>
          </item>
        </layout>
      </widget>
    </widget>
    <widget class="QMenuBar" name="menubar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>1042</width>
          <height>30</height>
        </rect>
      </property>
    </widget>
    <widget class="QStatusBar" name="statusbar"/>
  </widget>
  <customwidgets>
    <customwidget>
      <class>QVideoWidget</class>
      <extends>QWidget</extends>
      <header>QtMultimediaWidgets</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections/>
</ui>
